velox/substrait/SubstraitToVeloxPlan.cpp

```diff
@@ -340,11 +340,11 @@ core::PlanNodePtr SubstraitVeloxPlanConverter::toVeloxPlan(
       splitInfo->paths.emplace_back(file.uri_file());
       splitInfo->starts.emplace_back(file.start());
       splitInfo->lengths.emplace_back(file.length());
-      switch (file.format()) {
-        case 0:
+      switch (file.file_format_case()) {
+        case 11:
```

Comment:
Does Substrait provide constants for the values? Using raw numbers makes the code hard to read.

Reply:
I have used enums instead of numbers

Reply:
I have rebased onto latest master

Reply:
Thanks. format-check job is failing. Would you take a look?

Reply:
Thanks，I have formatted the code

Reply:
but, macos-build fail and test fail.

Reply:
Hmm... Make sure you are rebased onto lates main.

Reply:
Again there should be no need to make changes to the CMakelists file. Please reach out to me on slack @kgpai and maybe i can help you uncover why the brew install is failing for you @PenghuiJiao .

Reply:
Can you invite me to join your group on slack? @kgpai

Reply:
I couldnt add you myself . I have requested @jijufb to add you to Velox on slack.

Reply:
Invite is sent on Velox Slack Channel.